src/apiProxy/api-proxy.test.js

```diff
@@ -94,6 +95,17 @@ describe('buildRequestArgs', () => {
 		expect(postArgs.body).toEqual(jasmine.any(String));  // post requests will add body string
 		// post requests will add body string
 		expect(postArgs.headers['content-type']).toEqual('application/x-www-form-urlencoded');
+
+	});
+
+	const testQueryResults_utf8 = mockQuery(MOCK_RENDERPROPS_UTF8);
+	const apiConfig_utf8 = queryToApiConfig(testQueryResults_utf8);
+	const encoded_url = '/%E3%83%90-%E4%BA%AC';
+
+	it('Properly encodes the URL', () => {
+		const method = 'get';
+		const getArgs = buildRequestArgs({ ...options, method })(apiConfig_utf8);
+		expect(getArgs.url.split('?')[0]).toEqual(encoded_url);
```

Comment:
I think a slightly more general test would parse the URL for the `pathname` and then test that against an [isASCII regex](http://stackoverflow.com/a/14313213/579685)

```js
const { pathname } = require('url').parse(getArgs.url);
expect(/^[\x00-\xFF]*$/.test(pathname)).toBe(true);
```